CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## [2.1.0] TBD
+## [2.1.0] 2017-10-02
 ### Changed
+* Changed the default GC settings, tuning for our environment
```

Comment:
"our" environment doesn't mean much to anyone else, rather just describe what the defaults have changed to (less overall ram, more reserved percent) and/or what this change to the default settings gets a user

Reply:
good point, changed it